axis_backend/axis_backend/permissions/base.py

```
@@ -137,6 +137,61 @@ def has_object_permission(self, request, view, obj: Any) -> bool:
 
+class CanManageDocuments(permissions.BasePermission):
+    """
+    Permission for document management.
+
+    Use for operations that require document management privileges:
+    - Publishing/archiving documents
+    - Creating new versions
+    - Managing document lifecycle
+    """
+
+    def has_permission(self, request, view) -> bool:
```

Ruff (0.14.5)

Unused method argument: `view`

(ARG002)